cmd/node/main.go

```diff
@@ -52,13 +52,13 @@ import (
 )
 
 const (
-	defaultLogPath     = "logs"
-	defaultStatsPath   = "stats"
-	defaultDBPath      = "db"
-	defaultEpochString = "Epoch"
-	defaultShardString = "Shard"
-	metachainShardName = "metachain"
-	defaultRestApiPort = "off"
+	defaultLogPath         = "logs"
+	defaultStatsPath       = "stats"
+	defaultDBPath          = "db"
+	defaultEpochString     = "Epoch"
+	defaultShardString     = "Shard"
+	metachainShardName     = "metachain"
+	defaultRestApiInerface = "localhost:8080"
```

Comment:
Do we want to enable port 8080? Can the restApi be enabled just for localhost and not the outside?

Reply:
Yes, we will re-enable 8080. As found in this PR, one can configure also the interface on which the gin server is allowed to be bound on. localhost:8080 means that the server is able to open the port only on localhost interface (lan and wan connections are not possible).